README.md

```diff
@@ -9,8 +17,6 @@ See [docs/operations.md](docs/operations.md) for node operator instructions.
 
 ![](docs/images/overview.svg)
 
-⚠ **Wormhole v2 is in active development - see "main" branch for the v1 mainnet version** ⚠
-
 ### Audit / Feature Status
```

Comment:
There doesn’t seem to be anything related to this section, should we link to security.md and white papers? Or just take out the header.